jvcl/run/JvAni.pas

```diff
@@ -40,6 +40,9 @@ interface
   {$IFDEF UNITVERSIONING}
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
+  {$IFDEF HAS_UNIT_TYPES}
```

Comment:
No need for the IFDEF here, Types was introduced by Delphi 6 and we don't support anything earlier than Delphi 6